azavea/tilejson.io
# Add ability to hydrate state from GitHub Gist

## Overview

Upon sharing, a GitHub Gist is created. If the Gist ID is taken and appended to the url as such: `https://tilejson.io/g/:gistId`, the information in the gist will be used to hydrate the redux state and the map.

Connects #79 

## Testing Instructions

 * Throw in layers, play with base, opacity, visibility, etc. 
 * Share
 * Copy gistId and append to url `/g/:gistId`
 * Verify that center, zoom, map layers, base are all the same as when shared
